src/lib.rs

```diff
@@ -238,25 +254,24 @@ impl<'a> Server<'a> {
     }
 }
 
-pub fn download(remote_ssh_host : &str,
-                port_range      : connection::PortRange,
-                remote_path     : &str,
-                local_path      : PathBuf)
-{
+pub fn download(remote_ssh_host: &str,
+                port_range: connection::PortRange,
+                remote_path: &str,
+                local_path: PathBuf) {
```

Comment:
I <3 golangs I won't compile until you format the Rob Pike way.

Rust wanted to do that and just didn't have time before 1.0.